### PR TITLE
feat(api): move openUrl to common api provided by Einstein pluginHost

### PR DESCRIPTION
**Pull request by xatier**

### CODE REVIEW

File: src/pluginHost.node/api/methods/openUrl.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { EnvironmentPlatform, openUrlFn } from 'einstein'
+
+import { spawn } from './spawn'
+
+export const openUrl: openUrlFn = async (platform: EnvironmentPlatform, url: string) => {
```
Comment (davy-tw) [inline src/pluginHost.node/api/methods/openUrl.ts]: suggestion: platform can be fetched from `Environment` (`src/pluginHost.node/Environment.ts`) directly

Comment (xatier) [reply src/pluginHost.node/api/methods/openUrl.ts]: fixed.

**Comment by davy-tw** [comment]
suggestion: Squash commits?

**Comment by xatier** [comment]
> suggestion: Squash commits?

sure.

**Pull request merged**